tests/model_serving/model_server/private_endpoint/conftest.py

```diff
@@ -0,0 +1,191 @@
+import json
+import pytest
+from typing import Generator, Any
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.secret import Secret
+from ocp_resources.namespace import Namespace
+from ocp_resources.pod import Pod
+from simple_logger.logger import get_logger
+from ocp_resources.serving_runtime import ServingRuntime
+from kubernetes.dynamic import DynamicClient
+
+from utilities.serving_runtime import ServingRuntimeFromTemplate
+from tests.model_serving.model_server.utils import create_isvc
+from tests.model_serving.model_server.private_endpoint.utils import (
+    create_sidecar_pod,
+    get_flan_deployment,
+    b64_encoded_string,
+)
+from utilities.infra import create_ns
+from tests.model_serving.constants import KSERVE_SERVERLESS
+
+
+LOGGER = get_logger(name=__name__)
+
+
+@pytest.fixture(scope="session")
+def endpoint_namespace(admin_client: DynamicClient) -> Generator[Namespace, None, None]:
+    with create_ns(admin_client=admin_client, name="endpoint-namespace") as ns:
+        yield ns
+
+
+@pytest.fixture(scope="session")
+def diff_namespace(admin_client: DynamicClient) -> Generator[Namespace, None, None]:
```

Comment:
Another nitpick, but I don't like these `diff_*` fixture names; they are not very descriptive and what if you need two namespaces in another test again?

Reply:
Let's open an issue and think about this

Reply:
:+1:

Reply:
Opened https://github.com/opendatahub-io/opendatahub-tests/issues/45